shared/src/business/utilities/documentGenerators/constants/printableTrialSessionWorkingCopyConstants.js

```diff
@@ -0,0 +1,462 @@
+const FORMATTED_CASES = [
+  {
+    calendarNotes:
+      'Spicy jalapeno bacon ipsum dolor amet laborum nostrud eiusmod, ex drumstick quis eu dolore excepteur fatback. Qui sirloin andouille chuck. Jowl velit capicola quis elit meatloaf.',
```

Comment:
MEAT! 🍖 

we do have a folder `shared/src/test` that contains constants that end up being used for testing purposes, would this be a good place to put this file? though that folder could probably be renamed to something more descriptive than `test` haha